### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by domenic99**
Project created in api mode

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,42 +1,21 @@
-*.rbc
-capybara-*.html
-.rspec
-/log
-/tmp
-/db/*.sqlite3
-/db/*.sqlite3-journal
-/public/system
```
Comment (rakvium) [inline .gitignore]: @domenic99 why did you remove this string?

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  gem 'listen', '~> 3.0.5'
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem 'active_model_serializers'
```
Comment (rakvium) [inline Gemfile]: Please add a newline to the end of the file :construction_worker_man:

File: .gitignore (hunk 1 above)
Comment (domenic99) [inline .gitignore]: @rakvium it was made by rails ) but it wasn't removed, it was added in the end of file

Comment (rakvium) [inline .gitignore]: @domenic99 also I recommend to add string `*~` to `.gitignore` to ignore temporary files from editors.

**Comment by domenic99** [comment]
@rakvium Also I switched from sqltie to pg
